packages/checkout-com/src/helpers.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable */
+interface PaymentPropetiesWithoutToken {
+    context_id: string,
+    save_payment_instrument?: boolean,
+    secure3d?: boolean,
+    success_url?: string,
+    failure_url?: string
```

Comment:
just pass here `token?` as optional field and call this interface `PaymentPropeties` simpler.